.buildkite/steps/build-docker-image.sh

```diff
@@ -23,6 +23,7 @@ build_docker_image() {
   local packaging_dir="$2"
 
   echo "--- Building :docker: $image_tag"
+  cp -a packaging/linux/root/usr/share/buildkite-agent/hooks/ "${packaging_dir}/hooks/"
```

Comment:
I wonder if people look in the actual image for example hooks, or the Docker directories in GitHub? I wondered if the real problem was a lack of line to the standard example hooks from places like the Docker agent setup docs.

Reply:
Yeah, dunno, I've spoken to one human that did look for the example hooks there, so thought I'd do this. I can remove it if it's controversial.

Reply:
Yep, I remembered. I just wasn’t sure where they exactly went looking.